etc/picongpu/taurus-tud/knl.tpl

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# Copyright 2013-2017 Axel Huebl, Richard Pausch, Alexander Matthes
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# PIConGPU batch script for taurus' SLURM batch system
+
+#SBATCH -p !TBG_queue
+#SBATCH --time=!TBG_wallTime
+# Sets batch job's name
+#SBATCH --job-name=!TBG_jobName
+#SBATCH -N !TBG_nodes
+#SBATCH -c !TBG_coresPerAcc
+#SBATCH --mem=90000
```

Comment:
Is 90GB the available memory per node?

Reply:
[This documentation](https://doc.zih.tu-dresden.de/hpc-wiki/bin/view/Compendium/KnlNodes) says I should use `--mem=90000` so I do. However it works also without. More interesting would probably to force not to get an SNC or flat memory node.

Reply:
ok, but is `SBATCH --mem` the MEM per node or the overall memory of the requested job? You might need to multiply the value in here with `!TBG_nodes`

Reply:
This is per node

Reply:
My answer disappeared? O_o

Reply:
Ah, found it. Only a *bit* out of context now: https://github.com/ComputationalRadiationPhysics/picongpu/pull/2270#discussion_r140221593